dev/cognitiveservices/data-plane/Language/analyzeconversations.json

```diff
@@ -2210,6 +2219,52 @@
         }
       ]
     },
+    "ChapteringParameters":{
+      "type": "object",
+      "description": "The parameters to control the chaptering.",
+      "properties": {
+        "levels":{
+          "type": "integer",
+          "format": "int32",
+          "description": "The default value is 0. 0 level means only 1 summary is generated for the whole input and no chaptering is performed. 1 level means flat chapters. 2 levels means each high level chapter can have child low level sub chapters. In result, the summaries field is populated if level is 0 or no specified, and the chapters field is populated if level > 0."
+        },
+        "maxChapterLengthInCharacters":{
+          "type": "integer",
+          "format": "int32",
+          "description": "The maximum number of characters in each chapter. If not specified, the model will determine the chapter length smartly."
+        },
+        "minChapterLengthInCharacters":{
+          "type": "integer",
+          "format": "int32",
+          "description": "The minimum number of characters in each chapter. If not specified, the model will determine the chapter length smartly."
+        }
+      }
+    },
+    "Chapter": {
+      "type": "object",
+      "description": "A predicted chapter of the conversation. Only used for client application CTS, MCU, SCIS and MeTA.",
+      "properties": {
+        "chapterStartTimeTick": {
```

Comment:
Should we also include Duration or EndTime?

Reply:
EndTime is not needed. I added duration.